com.unity.visualeffectgraph/Editor/Compiler/VFXGraphCompiledData.cs

```diff
@@ -845,6 +845,25 @@ private void GenerateShaders(List<GeneratedCodeData> outGeneratedCodeData, VFXEx
                     descs[i].source = generated.content.ToString();
                     descs[i].name = fileName;
                     descs[i].compute = generated.computeShader;
+
+                    if (generated.computeShader)
```

Comment:
This (ln 849-866) will be removed after the C++ PR lands. It is a mechanism to support faux material serialization to unblock development and allow testing everything else.